Dockerfile

```diff
@@ -43,6 +43,13 @@ RUN yum install -y \
         gperftools-devel && \
     yum clean all --enablerepo=* && rm -rf /var/cache/yum/
 
+# If necessary, download the authfile-update version that can do per-instance authfiles
+RUN \
+    if /usr/local/bin/pkg-cmp-lt.sh xcache 3.2.0; then \
+        curl -Lo /usr/libexec/xcache/authfile-update https://raw.githubusercontent.com/opensciencegrid/xcache/V3-branch/src/authfile-update && \
```

Comment:
Right into /usr, ey?  Bold!

So xcache gets installed from yum out of `BASE_YUM_REPO`, but `authfile-update` is always taken from the tip of `V3-branch` (not necessarily part of a release) ?

Reply:
Yup. More specifically, `authfile-update` will be taken from the tip of `V3-branch` until `xcache-3.2.0` or newer is in `BASE_YUM_REPO`.

Reply:
Okie doke then!